src/lib/WIFI/devWIFI.cpp

```diff
@@ -796,6 +801,238 @@ static void WebUploadForceUpdateHandler(AsyncWebServerRequest *request) {
   }
 }
 
+#if defined(RADIO_LR1121)
```

Comment:
Can all of this code be moved to its own file, since it is all self contained (apart from reusing totalSize) and can be kept separate?